x-pack/plugin/esql/qa/testFixtures/src/main/resources/k8s-timeseries.csv-spec

```diff
@@ -784,5 +784,184 @@ max_deriv:double | max_rate:double | time_bucket:datetime     | cluster:keyword
 8.922491         | 11.56274        | 2024-05-10T00:10:00.000Z | prod
 16.62316         | 11.86081        | 2024-05-10T00:15:00.000Z | prod
 9.026268         | 6.980661        | 2024-05-10T00:20:00.000Z | prod
+;
+
+bare_count_over_time_outputs_dimensions
+required_capability: ts_command_v0
+required_capability: metrics_group_by_all
+  
+TS k8s
+| STATS count = count_over_time(network.cost)
+| SORT count DESC
+;
+
+count:long | _timeseries:keyword
+29         | KBGrBhmEnziumfgfOq1dn9NyLSzHck3A2MQiFrxrlvTQFP7YxbJ0rYg=
+29         | KBGrBhmEnziumfgfOq1dn9NyLSzHbdLj0kfy/m+tXh+yxVR6b3E17ss=
```

Comment:
Maybe add sorting by _timeseries, since the same counts can appear in any order.